### PR TITLE
Prep for 0.9.4 release

### PR DESCRIPTION
**Pull request by alexweininger**
We had to skip 0.9.3 because of a marketplace issue. I'm going to try to overwrite that release by bumping the version to 0.9.4.

### CODE REVIEW

**Comment by alexweininger** [comment]
Waiting for confirmation from Jason & co. before merging.

File: CHANGELOG.md
Comment (nturinski) [inline CHANGELOG.md:11]: Should we write a little note as to why it was skipped? It probably doesn't really matter, but might be nice to document for future us.

**Review by nturinski — APPROVED** [review]

**Review by nturinski — APPROVED** [review]

**Pull request merged**
